doc/api/diagnostics_channel.md

```diff
@@ -132,6 +132,35 @@ if (diagnostics_channel.hasSubscribers('my-channel')) {
 }
 ```
 
+#### `diagnostics_channel.hasChannel(name)`
```

Comment:
This is probably worth moving to its own PR.

Reply:
Probably, yes. Just was a tiny thing I needed here so added that quickly. 🙂